packages/cli/test/functional/test_site/testPanels.md

```diff
@@ -0,0 +1,55 @@
+**Panel with attributes**
+
+<panel header="Correct header">
+  Lorem ipsum...
+</panel>
+
+<br>
+
+<panel header="Correct header" alt="Alternative header">
+  Lorem ipsum...
+</panel>
+
+<br>
+
+**Panel with slots**
+
+<panel>
+  <span slot="header">Correct header</span>
+  Lorem ipsum...
+</panel>
+
+<br>
+
+<panel>
+  <span slot="header">Correct header</span>
+  <span slot="alt">Alternative header</span>
+  Lorem ipsum...
+</panel>
+
+<br>
+
+**Panel with slots overriding attributes**
+
+<panel header="Should not appear: Overwritten header">
+  <span slot="header">Correct header</span>
+  Lorem ipsum...
+</panel>
+
+<br>
+
+<panel alt="Should not appear: Overwritten alternative header">
+  <span slot="alt">Alternative header</span>
```

Comment:
We can remove the `alt`s in these tests since `alt` is not supported as a slot (see below)

Reply:
Also noted! I have removed this in [25f4250](https://github.com/MarkBind/markbind/pull/2053/commits/25f42501e8da056540cb26e7b51f23ce0746c88f) as well :+1: